docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,152 @@
+This can be done by fetching the last config block and making sure that on each channel the `Type` ia Raft, the `Metadata` matched the Raft metadata and the `State` is `STATE_MAINTENANCE`.
+
+If the channels have been updated successfully, the ordering service is now ready for backup.
+
+Note:
+While in maintenance mode, normal transactions, config updates unrelated to migration, and `Deliver` requests from the peers used to retrieve new blocks are rejected. 
+This is done in order to prevent the need to both backup, and if necessary restore, peers during migration, as they only receive updates once migration has successfully completed. 
+In other words, we want to keep the ordering service backup point, which is the next step, ahead of the peer’s ledger, in order to be able to perform rollback if needed. 
+However, ordering node admins can issue `Deliver` requests (which they need to be able to do in order to continue the migration process).
+
+
+## Phase 2 - backup 
+Shut down all Raft ordering nodes, create a backup of the file system of these servers. Then restart the ordering service nodes.
+
+
+## Phase 3 - switch to BFT in maintenance mode
+This step in the migration process is another channel configuration update for each channel. 
+In this configuration update, switch the `Type` to `BFT` while keeping the State in `STATE_MAINTENANCE`, 
+and fill in the `Metadata` configuration and the `ConsentersMapping`. 
+Keep to set the `Metadata` configuration identically on all channels and keep to set the `ConsenterMapping` with all orderers.
+
+Then, validate that each ordering service node has committed the `ConsensusType` and `Orderers` changes by pulling and inspecting the last config block on each channel.
+
+Note: For each channel, the transaction that changes the `ConsensusType` and `Orderers` must be the last configuration transaction before restarting the nodes in the next step. 
+If some other configuration transaction happens after this step, the nodes will most likely crash on restart, or result in undefined behavior.
+
+The log output that shows a migration is detected, for example in a cluster of 4 nodes and channel `testchannel`, is:
+```
+Detected migration to BFT channel=testchannel node=1
+Detected migration to BFT channel=testchannel node=2
+Detected migration to BFT channel=testchannel node=3
+Detected migration to BFT channel=testchannel node=4
+```
+
+## Phase 4 - restart and validate leader
+Note: exit of maintenance mode must be done after restart.
+
+After the configuration update from the previous step has been completed on each channel, restart the ordering service nodes. 
+They should restart as `BFT` nodes, form a cluster per channel, and elect a leader on each channel.
+
+The log output which proves that a BFT cluster is configured is: ```SmartBFT-v3 is now servicing chain testchannel channel=channel-name```
```

Comment:
Can we get/validate the consensus type and healthy status from restful API?

Reading log is not a good way here.

Reply:
The way to validate the type is to ask for the last block from all orderers and make a transaction after exiting maintenance mode. This can verify that each orderer is switched to BFT and working properly.
This way is generally offered in all stages of migration for validation purposes.